airflow/www/jest.config.js

```diff
@@ -32,6 +32,7 @@ const config = {
   transformIgnorePatterns: [
     `node_modules/(?!${[
       // specify modules that needs to be transformed for jest. (esm modules)
+      "ansi_up",
```

Comment:
Just opened conversation to keep it non-mergable until @jscheffl  fixes the glitch